### ISSUE TITLE
mae and mse loss definitions should use tf.reduce_mean

### ISSUE THREAD

**Issue by abhinav-upadhyay** [philipperemy/n-beats#2]
Hi

The `mae` and `mse` definitions in `model.py` use `tf.reduce_sum`, shouldn't they use `tf.reduce_mean`?

**Comment by philipperemy** [comment]
@abhinav-upadhyay yes that's correct. It does not change anything in practice I guess. So they are equal up to a scalar.

**Comment by abhinav-upadhyay** [comment]
Yes, that's true. I was trying it out it on the NYC taxi rides dataset and thought instead of training it with batch size 1, I will use a larger batch size where the mae loss was showing up as a large number. Having a mean value is more meaningful in that case :)

**Comment by philipperemy** [comment]
Oh yeah for sure :)

**Comment by philipperemy** [comment]
This is the branch where I'm working now:

https://github.com/philipperemy/n-beats/pull/3/files

It includes some fixes that I will later push to master once it's stable.

**Comment by philipperemy** [comment]
Done!

**Comment by abhinav-upadhyay** [comment]
Thanks :)
